src/cli/index.ts

```diff
@@ -63,6 +63,7 @@ function printUsage() {
 const signOptions = {
   oidcClientID: 'sigstore',
   oidcIssuer: 'https://oauth2.sigstore.dev/auth',
+  oidcRedirectURL: process.env.OIDC_REDIRECT_URL,
```

Comment:
Should we prefix the env var with `SIGSTORE`?

```suggestion
  oidcRedirectURL: process.env.SIGSTORE_OIDC_REDIRECT_URL,
```